Sources/SwiftInspectorVisitors/ClassVisitor.swift

```diff
@@ -52,11 +52,8 @@ public final class ClassVisitor: SyntaxVisitor {
       // Base case. We've previously found a class declaration, so this must be an inner class.
       // This class visitor shouldn't recurse down into the children.
       // Instead, we'll use a new class visitor to get the information from this class.
-      let qualifiedParentTypeName = QualifiedParentNameCreator.createNameGiven(
```

Comment:
`QualifiedParentNameCreator` was deleted in favor of a `createTypeWithName(_:parent:)` method on `TypeDescription`. This change was made everywhere.